app/services/user.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcrypt');
+const { Sequelize, ...db } = require('../models');
+const errors = require('../errors');
+
+const { Op } = Sequelize;
+
+const toEncrypt = data => {
+  const saltRounds = 5;
+  const salt = bcrypt.genSaltSync(saltRounds);
+  const hash = bcrypt.hashSync(data, salt);
+  return hash;
+};
+
+exports.verifyUniqueEmail = async email => {
+  try {
+    const emailUser = await db.User.findOne({
+      where: {
+        email: {
+          [Op.eq]: email
+        }
+      }
+    });
+    return emailUser !== null;
+  } catch {
```

Comment:
Remember the handling of the try-catch, as it was done in the weet service, the catch must have the error variable and then make a log of it.